### ISSUE TITLE
jar files on load_path do not obey Dir.pwd

### ISSUE THREAD

**Issue by mkristian** [jruby/jruby#2215]
this fails
```
cd bin
jruby -I. -e "Dir.chdir('..'); require 'test/test_jar_on_load_path.rb'"
```

but this passes
```
bin/jruby -I. -e "require 'test/test_jar_on_load_path.rb'"
```

JarResource does not take CWD (or PWD) into account and uses the current directory of the JVM instead.

**Comment by rtyler** [comment]
@mkristian I'm pretty sure you've fixed this in recent versions of JRuby 1.7.x right?

**Comment by mkristian** [comment]
nope - I think I was on the wrong track here. behaviour as is seems correct. closing
